modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/portlet/action/EditDepotEntryMVCActionCommand.java

```diff
@@ -120,6 +121,8 @@ private void _updateDLSizeLimitConfiguration(
 
 		Map<String, String[]> parameterMap = actionRequest.getParameterMap();
 
+		long fileMaxSize = MapUtil.getLong(parameterMap, "fileMaxSize", 0L);
+
```

Comment:
You should move this up before the `mimeTypeSizeLimits` declaration (in general, variable declaration should follow the order in which they're used as parameters). Alternatively, and if the auto sf doesn't complain, you may inline the variable.

Reply:
Updated, And as you expected SF accepted the inlined variable.